users/views.py

```diff
@@ -72,3 +72,37 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+class KakaoSignInView(View):
+    def post(self, request):
+
+        try:
+            data            = json.loads(request.body)
+            profile_request = requests.get(
+                "https://kapi.kakao.com/v2/user/me",
+                headers = {"Authorization" : f"Bearer {data['access_token']}"}
+            )
+
+            profile_json  = profile_request.json()
+            kakao_account = profile_json.get("kakao_account", None)
+            if kakao_account == None:
```

Comment:
```suggestion
            if not kakao_account:
```
변수가 `None`이 아니라면 if not 을 했을 대 `True`가 나오게 됩니다. 이렇게 바로 True / False 값을 바로 평가하지 않을 때는 연산자를 사용하지 않는 것이 보다 파이썬스러운 코드입니다 :)

Reply:
감사합니다. 수정하였습니다.